packages/ui/components/VisualEditor.tsx

```diff
@@ -43,6 +37,14 @@ export const VisualEditor: React.FC<VisualEditorProps> = ({ schema, onSchemaChan
     onSchemaChange(newSchemaString);
   };
 
+  const selectStyle = {
```

Comment:
Instead of styling the select like this, I suggest we use the `DropDownMenu` item that we already have in the our `ui` package. the `trigger` can also be what is already in the figma design easily.